examples/micronaut/src/main/java/example/micronaut/HelloController.java

```diff
@@ -0,0 +1,13 @@
+package example.micronaut;
```

Comment:
We'd prefer to keep the Groovy HelloController since this example is also meant to show Jib containerizing a Groovy app. Feel free to include both the Java and Groovy implementations though.